content/hi/docs/reference/glossary/kube-controller-manager.md

```diff
@@ -0,0 +1,17 @@
+---
+title: kube-controller-manager
```

Comment:
This needs to be in Hindi

Reply:
You can use `aka:` to also provide the original name.